CMakeLists.txt

```diff
@@ -15,6 +15,8 @@
 #
 #######################
 
+set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O3 -g -DNDEBUG")
```

Comment:
Could we add a comment explaining why we introduced this? I'm familiar with how these seemingly little, insignificant settings accumulate in the build until nobody can decipher it anymore.